plugins/chat/app/jobs/regular/chat_notify_watching.rb

```diff
@@ -43,7 +43,7 @@ def execute(args = {})
     def send_notifications(membership)
       user = membership.user
       guardian = Guardian.new(user)
-      return unless guardian.can_chat? && guardian.can_see_chat_channel?(@chat_channel)
+      return unless guardian.can_chat? && guardian.can_join_chat_channel?(@chat_channel)
```

Comment:
no tests need to change or be created for this?

Reply:
Because guardian is tested, I added a test which ensures that guardian is invoked